current_version.txt

```diff
@@ -1 +1 @@
-1.7.0
+1.6.1
```

Comment:
What is the reason for this downgrade?

Reply:
It will cause the full suite of tests to be retriggered for 1.7.0 using `resource` and generating .svg files to create a working README